README.rst

```diff
@@ -14,6 +14,8 @@ Install MLflow from PyPi via ``pip install mlflow``
 
 MLflow requires ``conda`` to be on the ``PATH`` for the projects feature.
 
+Nightly snapshots of MLflow master are also available `here <https://mlflow-snapshots.s3-us-west-2.amazonaws.com/>`_.
```

Comment:
Could we also document how to construct the URL of a wheel (as in your PR description) here? Just visiting https://mlflow-snapshots.s3-us-west-2.amazonaws.com/ leads to an error message page for me

Reply:
We may also be able to change permissions on this bucket to allow listing.

Reply:
That's not an error page. That's the list.

Reply:
Ah my mistake...this looks good then, thanks Aaron!